source/scripts/script.js

```diff
@@ -1,7 +1,20 @@
 // Import API from file
 import { fetchRecipes } from "./api_script.js";
+import { Router } from './Router.js';
```

Comment:
Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/cse110-fa21-group23/cse110-fa21-group23/pullRequest?prid=8422335)
```suggestion
import { Router } from "./Router.js";
```